README.md

```diff
@@ -4,7 +4,54 @@
 
 ## Overview
 
-*To be written*
+Agentmon reads language metrics from Prometheus or a StatsD source and posts them to the language metrics sink, for 
```

Comment:
Questions I have:

At a high level why is agentmon necessary? My understanding is that for Node agentmon will be used to consolidate metrics from multiple processes. From the description I assume that it allows you to re-use existing Prometheus or StatsD metrics libraries that report in that format?

How do I get Agentmon onto a dyno? Should I add it to a Node buildpack?

How can I get this running locally and run tests if I want to make a change?